### PR TITLE
Cleanup workflows

### PR DESCRIPTION
**Pull request by ncfavier**
The install-nix-action now installs a Flakes-enabled Nix by default. I also removed the call to jq because I think it's currently just obscuring an error.

### CODE REVIEW

**Comment by ncfavier** [comment]
Indeed the index names were wrong. I reintroduced jq and made it print the whole response if `took` is missing.

**Comment by ncfavier** [comment]
Everything looks fine now: [frontend](https://github.com/NixOS/nixos-search/runs/4832344257?check_suite_focus=true), [flakes](https://github.com/NixOS/nixos-search/runs/4832344258?check_suite_focus=true), [nixpkgs](https://github.com/NixOS/nixos-search/actions/runs/1704348247).

File: .github/workflows/import-nixpkgs.yml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: true
+      matrix:
+        channel:
+          - unstable
+          - 21.11
+          - 21.05
+          - 20.09
```
Comment (garbas) [inline .github/workflows/import-nixpkgs.yml:21]: We should probably retire `20.09` channel.

Comment (ncfavier) [reply .github/workflows/import-nixpkgs.yml:21]: I was thinking the same since it's EOL

**Review by garbas — APPROVED** [review]
:+1:

File: .github/workflows/import-nixpkgs.yml (hunk 1 above)
Comment (ncfavier) [reply .github/workflows/import-nixpkgs.yml:21]: Probably belongs in a separate PR though

**Comment by ncfavier** [comment]
I will!

**Pull request merged**